newdle/api.py

```diff
@@ -191,6 +191,19 @@ def update_newdle(args, code):
     return NewdleSchema().jsonify(newdle)
 
 
+@api.route('/newdle/<code>/participants/me')
+@allow_anonymous
+def get_participant_me(code):
+    participant = Participant.query.filter(
+        Participant.newdle.has(Newdle.code == code),
+        Participant.auth_uid == g.user['uid'],
+    ).first()
+    if participant:
+        return ParticipantSchema().jsonify(participant)
+    else:
+        return jsonify(None)
```

Comment:
@ThiefMaster Returning None in this case breaks the answer reducer (as this is returned from `PARITICPANT_RECIEVED` and it's null)

Reply:
I don't think so. I'll add a check  for this.